recipes/cppbenchmark/all/CMakeLists.txt

```
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.13)
+project(cmake_wrapper)
+
+include(${CMAKE_CURRENT_BINARY_DIR}/../conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(WIN32)
+    add_definitions (-D_CRT_SECURE_NO_WARNINGS)
+endif()
+
+enable_testing()
+
+add_subdirectory("source_subfolder")
+
+conan_target_link_libraries(cppbenchmark)
+
+target_compile_features(cppbenchmark PUBLIC cxx_std_17)
+if(CPPBENCHMARK_TESTS)
+    target_compile_features(cppbenchmark-tests PUBLIC cxx_std_17)
+endif()
+
+install(DIRECTORY source_subfolder/include/
+        DESTINATION include
+        FILES_MATCHING
+        PATTERN "*.h"
+        PATTERN "*.inl")
+
+if(UNIX AND NOT APPLE)
+    target_link_libraries(cppbenchmark pthread)
+endif()
+
+if(WIN32)
+    # target_link_libraries(cppbenchmark rpcrt4 userenv)
+    if(BUILD_SHARED_LIBS)
+        set_target_properties(cppbenchmark PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
+    endif()
+endif()
```

```suggestion
if(UNIX AND NOT APPLE)
    target_link_libraries(cppbenchmark pthread)
endif()

if(WIN32)
    # target_link_libraries(cppbenchmark rpcrt4 userenv)
    if(BUILD_SHARED_LIBS)
        set_target_properties(cppbenchmark PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
    endif()
endif()
```

You could send it as a PR to the upstream. It seems a valid patch.

---

Sorry, I overlooked and the project actually already handles the thread libraries with:

```cmake
find_package(Threads REQUIRED)
# ...
list(APPEND LINKLIBS Threads::Threads)
# ...
target_link_libraries(cppbenchmark ${LINKLIBS})
```

Fixed in 7db23f01